Align protocol docs with style guide (#575)
LiskHQ/lisk-docs: modules/ROOT/pages/consensus-algorithm.adoc

= Consensus algorithm

> :description: The consensus algorithm section of the Lisk protocol illustrates how the delegates eligible to create blocks are selected and how they reach agreement on new blocks.

Reviewer: ```suggestion
:description: This section illustrates how the delegates eligible to create blocks are selected and how they reach agreement on new blocks.
```
